src/sql/workbench/contrib/editData/browser/gridParentComponent.ts

```diff
@@ -380,17 +381,20 @@ export abstract class GridParentComponent {
 	}
 
 	openContextMenu(event, batchId, resultId, index): void {
-		let slick: any = this.slickgrids.toArray()[index];
-		let grid = slick._grid;
+		//let slick: any = this.slickgrids.toArray()[index];
```

Comment:
please remove all dead code from all files prior to sending for final PR review. Thanks!